phylogenetic/rules/annotate_phylogeny.smk

```diff
@@ -80,14 +92,20 @@ rule traits:
         columns = config["traits"]["columns"],
         sampling_bias_correction = config["traits"]["sampling_bias_correction"],
         strain_id = config.get("strain_id_field", "strain"),
+    log:
+        "logs/traits.txt",
+    benchmark:
+        "benchmarks/trais.txt"
     shell:
-        """
+        r"""
+        exec &> >(tee {log:q})
+
         augur traits \
-            --tree {input.tree} \
-            --metadata {input.metadata} \
-            --metadata-id-columns {params.strain_id} \
-            --output {output.node_data} \
-            --columns {params.columns} \
+            --tree {input.tree:q} \
+            --metadata {input.metadata:q} \
+            --metadata-id-columns {params.strain_id:q} \
+            --output {output.node_data:q} \
+            --columns {params.columns:q} \
```

Comment:
Ah, the `:q` in `params.columns:q` turns into `--columns "region country"`. 

I would leave the `--columns {params.columns}`, I ran into this during the [mumps pipeline](https://github.com/nextstrain/mumps/commit/21237d1873acd9f12a15426e620ac42fa27894da).

Reply:
> Thoughts on keeping this backwards compatible by automatically splitting the param values if it's a string?

Sure, seems easy with a helper function that can be applied to both. Or, we could add an entry to changelog, or both?

Reply:
Updated phylo config with backwards compatibility in https://github.com/nextstrain/zika/pull/84/commits/11d2644441f4143db0bc560cf9b6b71ad53f4446 and updated changelog in [62d4608](https://github.com/nextstrain/zika/pull/84/commits/62d4608a88bb147bff794a468821a68a8d899cb9). 

(Also filled in changelog with user facing changes over the last ~year in [f8e94b5](https://github.com/nextstrain/zika/pull/84/commits/f8e94b5a27d109446864aaa9241c8c26d9b90563))